src/features/farming/mail/components/Inbox.tsx

```diff
@@ -15,9 +15,15 @@ interface Props {
   onRead: (index: number) => void;
 }
 
+const outerPanelMaxHeight = 39;
+
 export const Inbox: React.FC<Props> = ({ inbox, isLoading, onRead }) => {
+  const accordionBodyMaxHeight = Math.max(
+    outerPanelMaxHeight - 1.5 - (inbox.length + 1) * 3.5,
```

Comment:
Can you please explain the logic behind these operations?

Reply:
![image](https://user-images.githubusercontent.com/107602352/194836710-77540a46-18ea-4d7f-a2d6-4a9685acb44c.png)
The height of each inbox header is roughly 3.5.  There are `inbox.length` inbox headers and 1 announcement header.  I also added a 1.5 height buffer for border to make sure it is not overflowing.

It is probably not an elegant solution as I am not good at front end styling (eg. there might be better ways to fill content to parent max height without doing the calculations).  Any suggestions would be appreciated.

Reply:
I had a look at this issue and I believe that by putting overflow-y:scroll at parent should resolve the issue. Can you please try that?

Reply:
Thanks for your suggestion, however I am not able to wrap the announcement content in the parent correctly by setting overflow-y:scroll at parent (if the max height is set to infinite instead of accordionBodyMaxHeight in the accordion body

Reply:
If there isn't an elegant solution yet, I can hardcode the height to 27rem and open another ticket to improve it if one can be found

Reply:
@Polysemouse ,let me help you with that. 

I would suggest you to remove all the calculations and reinstate max height on outer-panel and accordion body. Then on the outer panel in this component add classes 'overflow-y-scroll bg-clip-padding' and add borderRadius: 0 in the style. Something like in the image below. This will help us to resolve the issue. I tried this on my local and it seems to be working fine. Please let me know if you face any issues. 
<img width="616" alt="image" src="https://user-images.githubusercontent.com/35486909/196020190-21dbfc5f-32cc-49dc-9874-ffcbaa83dffc.png">

Reply:
I tried implementing the fix (`relative max-h-[39rem] overflow-y-scroll bg-clip-padding` with style `borderRadius` = 0) you have suggested to `OuterPanel`.  If I keep the Accordion.Body height untouched (`max-h-[30rem]`), there are double scrolling bars:
![image](https://user-images.githubusercontent.com/107602352/196025028-ae00830f-3d3e-4c30-a81a-ad20dd56d051.png)

If I remove the Accordion.Body height (remove `max-h-[30rem]` from `className`), the previous accordion headers are scrolled outside of view:
![image](https://user-images.githubusercontent.com/107602352/196025153-33ff45aa-2270-430f-96de-50680b1c5833.png)

If I set the Accordion.Body height to `max-h-[27rem]`, the view looks fine but with an extra unscrollable bar.  However it defeats the purpose of `max-h-[39rem] overflow-y-scroll bg-clip-padding` at `OuterPanel` because the inner content height never exceed  `max-h-[39rem]` for the `OuterPanel`:
![image](https://user-images.githubusercontent.com/107602352/196025273-e2a3ce6e-62f6-40b7-911f-9f8829984b0b.png)

@HBandesh what do you think?